### ISSUE TITLE
Hello IBLISSIN.

### ISSUE THREAD

**Issue by PlavnikHacks** [iBLiSSINonYT/survivio-plus#2]
Prepare for forced warfare. You cannot steal cheats and think you get away with it. We are coming, either delete your cheat now or prepare for your last days.

**Good Luck.**

**Comment by iBLiSSIN** [comment]
LoL. Where is iBLiSSIN. Note : I am not steal this cheat, I am a fan of iBLiSSIN but I do not know where is him. I want continue his hack, if u want I delete it, I ready

**Comment by iBLiSSIN** [comment]
Oh, forgot talk to you. The survivio-plus hack is yours?

**Comment by PlavnikHacks** [comment]
You might want to delete this before you end up like IBLISSIN. We are the reason he quit.

**Comment by iBLiSSIN** [comment]
How scary are you that could make iBLiSSIN quit? Answer all my questions and I will delete it.

**Comment by iBLiSSIN** [comment]
How to you make him quit?

**Comment by iBLiSSIN** [comment]
Ha ha, i know you. You have in the video of Humphrey gaming. The iBLiSSIN said this hack is make by reuke and kalaborative and him continue this hack. You have the “very-good-surviv.io-hack” but you did delete it

**Comment by iBLiSSIN** [comment]
So who is this hack from? And who can continue it? iBLiSSIN or you?

**Comment by iBLiSSIN** [comment]
You didn't even continue to hack this and assign the repo to Humphrey Gaming. Why did you break iBLiSSIN when he did so well? If your reasons are not valid, I declare that I will never give up on iBLiSSIN's responsibility and will become a second iBLiSSIN. OK?

**Comment by iBLiSSIN** [comment]
IceHacks is really good but never will be as hacked by iBLiSSIN. It should have better aimbot, hack zoom. Humphrey Gaming is very good, but he never refuses to improve icehacks. Why could you find me so early?

**Comment by iBLiSSIN** [comment]
What if I told all that the person who made iBLiSSIN disappear are you?

**Comment by iBLiSSIN** [comment]
Hate you x99999999999999999

**Comment by PlavnikHacks** [comment]
He stole code, begged people to update it, sold his cheat for $5 that was totally free if you looked it up. Didn't even try to hide the face he stole code. And if it makes you happy IceHacks is coming back to work on his cheat.

Stop skidding kid. You will just end up like IBLISSIN.

**Comment by iBLiSSIN** [comment]
haha. ok i will delete it, But i will do for icehacks. For better aimbot

**Comment by russyboy** [comment]
> He stole code, begged people to update it, sold his cheat for $5 that was totally free if you looked it up. Didn't even try to hide the face he stole code. And if it makes you happy IceHacks is coming back to work on his cheat.
> 
> Stop skidding kid. You will just end up like IBLISSIN.

Yes icehacks is working on survivcheat injector 3.0 with scroll zoom

**Comment by russyboy** [comment]
icehacks is coming back this february( I think )

**Comment by iBLiSSIN** [comment]
haha

**Comment by iBLiSSIN** [comment]
The person tell for ice hacks know this idea is me

**Comment by 068scar** [comment]
Does this hack work I hope it doesn’t?

**Comment by 068scar** [comment]
> ice hacks is coming back this February( I think )

He will define to not and I wish he doesnt

**Comment by 068scar** [comment]
Also I bliss in the real one killed me yesterday with hacks and got 15 kills I think he is back :(

**Comment by 068scar** [comment]
> > He stole code, begged people to update it, sold his cheat for $5 that was totally free if you looked it up. Didn't even try to hide the face he stole code. And if it makes you happy IceHacks is coming back to work on his cheat.
> > Stop skidding kid. You will just end up like IBLISSIN.
> 
> Yes icehacks is working on survivcheat injector 3.0 with scroll zoom

100% sure it will not work

**Comment by russyboy** [comment]
should this topic be the legacy of Iblissin and his return

> The person tell for ice hacks know this idea is me
